### PR TITLE
Update docs to reflect production setup

### PR DESCRIPTION
**Pull request by Jkrzy**
Working towards resolution of #176

### CODE REVIEW

**Comment by Jkrzy** [comment]
@porta-antiporta looking good, thanks for jumping in!

The first column of each table here has no bottom-border, is that intentional?

File: docs/updating-uskpa-docs.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+## Application monitoring for the production instance
+[:arrow_left: Back to USKPA
+Documentation](../docs)
+
+This is only relevant to administrators of the USKPA site.
+
+### IRS Form 990
+The USKPA, being a non-profit organization, files an IRS Form 990 every year.  To meet the requirement in making these documents available to the public, the last 3 years of filings are posted to the uskpa.org site.  As such, each year the site needs to include the new filings. This change, along with all others to the site, must be committed to this
```
Comment (porta-antiporta) [inline docs/updating-uskpa-docs.md]: Committed this verbiage change so USKPA knows to keep those forms updated.

**Comment by porta-antiporta** [comment]
See above for minor comments.  But 👍 :shipit:

File: docs/deploy.md [hunk 2]
```diff
@@ -43,11 +43,11 @@ $ heroku run python manage.py migrate
 ```
 
 ### Production
-*Heroku app: TBD*
+*Heroku app: uskpa-prod*
 
-Deployed from: Tagged release
+Deployed from: Github [master] branch
```
[the remark this answers is not in the document]
Comment (Jkrzy) [reply docs/deploy.md]: Deploying a change is covered in  -- [change-workflow.md](https://github.com/USKPA-dev/uskpa/pull/177/files#diff-4090635743c9ba702ff1416e0c7a7bf0)

Will add a reference and link here.

**Comment by jastrowinkle** [comment]
Your assistance in dismantling the agency for national security issues is remarkable

**Pull request merged**